hash-parser/src/modules.rs

```diff
@@ -30,9 +45,134 @@ pub struct Modules {
     contents: Vec<String>,
 }
 
+/// @Incomplete: This will have to change given the fact that we  want to generate this information at compile time.
+///              Ideally, we want [`Self::get_stdlib_modules()`] to only generate a vector of pathbufs and the use
+///              that to resolve module paths.
 impl Modules {
     /// Get the module at the given index.
     pub fn get_module(&self, idx: ModuleIdx) -> Module<'_> {
         Module { idx, modules: self }
     }
+
+    /// Function that builds a module map of the standard library that is shipped
+    /// with the compiler distribution. Standard library modules are referenced
+    /// within imports
+    pub fn get_stdlib_modules(&self, dir: impl AsRef<Path>) -> Vec<PathBuf> {
```

Comment:
I thought it should panic since if we aren't able to read the standard library folder, it's in a pretty bad state, and we're likely unable to continue in the pipeline.